src/network/clientpackethandler.cpp

```diff
@@ -410,10 +410,10 @@ void Client::handleCommand_Movement(NetworkPacket* pkt)
 	Player *player = m_env.getLocalPlayer();
 	assert(player != NULL);
 
-	float mad, maa, maf, msw, mscr, msf, mscl, msj, lf, lfs, ls, g;
+	float mad, maa, maf, msw, mscr, msf, mscl, msj, lf, lfs, ls, g, dmg;
 
 	*pkt >> mad >> maa >> maf >> msw >> mscr >> msf >> mscl >> msj
-		>> lf >> lfs >> ls >> g;
+		>> lf >> lfs >> ls >> g >> dmg;
```

Comment:
Warn, you change the protocol, it's not allowed at this moment. Please use a compat protocol and wait for the new protocol to be designed